master/buildbot/test/unit/test_secret_in_file.py

```diff
@@ -35,11 +35,12 @@ def createTempDir(self, dirname):
         return tempdir.path
 
     def createFileTemp(self, tempdir, filename, text=None):
+        if text is None:
```

Comment:
this initialization pattern is when you initialize to a list of dict, if you initialize to a string(immutable), this is better to initialize in the default method parameter